app/javascript/gobierto_visualizations/webapp/visualizations/treeMapNested.vue

```diff
@@ -524,21 +546,37 @@ export default {
       }
 
       function breadcrumbs(d) {
-          let res = "";
-          const sep = ` <i style="display: inline-block; vertical-align: middle;" class="fas fa-angle-right"></i> `;
-          d.ancestors().reverse().forEach(({ data }) => {
-            if (res.includes('<b>')) {
-              res = res.replace(/<b>/g, '').replace(/<\/b>/g, '')
-            }
-            res += `<span><b>${data.name}</b></span>` + sep;
-          });
-          return res.split(sep).filter(i => i !== "").join(sep);
-        }
+        let res = "";
+        const sep = ` <i style="display: inline-block; vertical-align: middle;" class="fas fa-angle-right"></i> `;
+        d.ancestors().reverse().forEach(({ data }) => {
+          if (res.includes('<b>')) {
+            res = res.replace(/<b>/g, '').replace(/<\/b>/g, '')
+          }
+          res += `<span class="treemap-nested-breadcrumb"><b>${data.name}</b></span>` + sep;
+        });
+        return res.split(sep).filter(i => i !== "").join(sep);
+      }
+
+      function checkSizeBreadcumbs() {
+        const treeMapSidebar = document.querySelector('.treemap-nested-sidebar');
+        const treeMapSidebarWidth = treeMapSidebar.offsetWidth
+
+        const treeMapSidebarButton = document.querySelector('.treemap-nested-sidebar-button-group');
+        const treeMapSidebarButtonWidth = treeMapSidebarButton.offsetWidth
+
+        const treeMapAvailableWidth = treeMapSidebarWidth - treeMapSidebarButtonWidth
+
+        const treeMapSidebarNav = document.querySelector('.treemap-nested-sidebar-nav');
+        const treeMapSidebarNavWidth = treeMapSidebarNav.offsetWidth
+
+        const treeMapSidebarNavContainer = document.querySelector('.treemap-nested-sidebar-nav-breadcumb');
+        if (treeMapSidebarNavWidth > treeMapAvailableWidth) treeMapSidebarNavContainer.classList.add('ellipsis')
+      }
```

Comment:
Don't you think you use so many variables only to add a class?

Reply:
Done!